package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "weather-app",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build": "webpack --watch",
+    "dev": "webpack --mode development",
+    "build-production": "webpack --mode production",
+    "start": "webpack-dev-server --mode development --open"
```

Comment:
change `webpack-dev-server` to `webpack-serve` since this command was deprecated and no longer works. Also, ensure that `webpack-dev-server` is installed under the `devDependencies`

Reply:
In case perhaps the reviewer missed it, the user is able to change the metric from Celsius to  Fahrenheit and vice-versa by clicking on their symbols, these two clickable elements are working as a toggle button.